src/shell.ts

```diff
@@ -15,6 +15,7 @@ export class Shell {
     activeForm: Form | null
     commands: Map<string, Command>
     currentLine = ''
+    timer: NodeJS.Timer
```

Comment:
NodeJS ? It's strange as its a backend library. Find a cleaner way to satisfy the linter. Try [here](https://developer.mozilla.org/en-US/docs/Web/API/setInterval#return_value).